command/bridge/common/params.go

```diff
@@ -206,34 +206,20 @@ func ExtractExitEventIDs(receipt *ethgo.Receipt) ([]*big.Int, error) {
 // ExtractChildTokenAddr extracts predicted deterministic child token address
 func ExtractChildTokenAddr(receipt *ethgo.Receipt, childChainMintable bool) (*types.Address, error) {
```

Comment:
Remove unused param.
```suggestion
func ExtractChildTokenAddr(receipt *ethgo.Receipt) (*types.Address, error) {
```